mysite/twitterclone/models.py

```diff
@@ -1 +1,18 @@
 from django.db import models
+from django.urls import reverse
+from django.contrib.auth.models import User
+
+
+class Tweets(models.Model):
```

Comment:
このクラスで生成することが出来るインスタンスは一つのツイートを表現するので、単数形でクラス名を記述することをお勧めします。
```suggestion
class Tweet(models.Model):
```

Reply:
単数形に直しました。Model以外にもFormとhtmlも単数形に修正しています。